Modelica/ComplexBlocks/Examples/ShowTransferFunction/creation.txt

```diff
@@ -1,49 +1,65 @@
 [ResultCreationLog]
-modelName="Modelica.ComplexBlocks.Examples.ShowTransferFunction"
+modelName = "Modelica.ComplexBlocks.Examples.ShowTransferFunction"
 
 // Test info
-generationTool="Dymola Version 2020x (64-bit), 2019-10-10"
-generationDateAndTime="2020-05-05T00:31:42Z"
-gitURL="https://github.com/modelica/ModelicaStandardLibrary.git"
-gitRevision=bd753330a
-gitStatus="D Modelica/Resources/C-Sources/ModelicaInternal.c, M ModelicaServices/package.mo"
-testPC="LAPTOP"
-testOS="Microsoft Windows [Version 10.0.18363.815]"
-testUser="tbeu"
-testDescription="Reg test MSL v4.0.0-rc.1"
+generationTool = Dymola Version 2023x Refresh 1, 2023-04-12
+gitURL = https://github.com/modelica/ModelicaStandardLibrary
+gitRevision = 677bec0b
+gitStatus = M Resources/Library/win32/ModelicaIO.lib
+ M Resources/Library/win32/ModelicaMatIO.lib
+ M Resources/Library/win32/ModelicaStandardTables.lib
+ M Resources/Library/win32/gcctdm510/libModelicaIO.a
+ M Resources/Library/win32/gcctdm510/libModelicaMatIO.a
+ M Resources/Library/win32/gcctdm510/libModelicaStandardTables.a
+ M Resources/Library/win32/gcctdm510/libzlib.a
+ M Resources/Library/win32/zlib.lib
+ M Resources/Library/win64/ModelicaIO.lib
+ M Resources/Library/win64/ModelicaMatIO.lib
+ M Resources/Library/win64/ModelicaStandardTables.lib
+ M Resources/Library/win64/gcctdm510/libModelicaIO.a
+ M Resources/Library/win64/gcctdm510/libModelicaMatIO.a
+ M Resources/Library/win64/gcctdm510/libModelicaStandardTables.a
+ M Resources/Library/win64/gcctdm510/libzlib.a
+ M Resources/Library/win64/zlib.lib
+testPC = LTXA320
+testOS = Windows [Version 10.0.19041]
+testUser = Matthias Schaefer
+testDescription = 
+compiler = Microsoft Visual Studio 2022
+generationDateAndTime = 2024-03-18 23:58:18
 
 // Experiment settings (standardized annotation)
-StartTime=0
-StopTime=1 // from model
-Interval=0.0005 // used annotation from model, multiplied by 0.5
-Tolerance=1e-06 // used default, because no tolerance annotation in model
+StartTime = 0.0    // set by user
+StopTime = 1.0    // from model. experiment annotation or dymola default
+Interval = 2000    // from model. experiment annotation or dymola default
```

Comment:
This is incorrect (and similar for other update results).

The model experiment annotation has `Interval=0.001`. This seems more like a "number of steps", rather than `Interval`.

The older creation.txt also had different comments that described how the settings were decided, while this sounds like it has the same comment regardless of how it was decided?

Reply:
Thanks for your feedback. 

The pullrequest was reset to the draft-status, because of these inconsistencies in the NumberOfIntervals\ OutputInterval. 

The creation.txt and so also the comments are generated automatically. So we don't know the intension behind the settings. With the comment we only mark that the simulation setting correspond with the experiment annotation or not.

Reply:
> With the comment we only mark that the simulation setting correspond with the experiment annotation or not.

But with the current comment I cannot tell if it came from the experiment annotation or from the dymola default?